client/config/peerhost.go

```diff
@@ -238,6 +238,7 @@ type DownloadOption struct {
 	TotalRateLimit       util.RateLimit    `mapstructure:"totalRateLimit" yaml:"totalRateLimit"`
 	PerPeerRateLimit     util.RateLimit    `mapstructure:"perPeerRateLimit" yaml:"perPeerRateLimit"`
 	PieceDownloadTimeout time.Duration     `mapstructure:"pieceDownloadTimeout" yaml:"pieceDownloadTimeout"`
+	GRPCDialTimeout      time.Duration     `mapstructure:"grpcDialTimeout" yaml:"grpcDialTimeout"`
```

Comment:
why not GRPCTimeout?

Reply:
It is not easy to pick a timeout for bidirection grpc timeout, but dial timeout is okay.